static/styles/compose.css

```diff
@@ -177,6 +177,7 @@
     display: none;
     position: absolute;
     right: 0;
+    opacity: 1;
```

Comment:
Can you post a screenshot?  We probably want the same color as the compose_control_button icons; I'm not sure if this ends up being brighter.

Reply:
https://github.com/zulip/zulip/pull/18154#issuecomment-820336359

Reply:
<img width="1060" alt="Screenshot 2021-04-21 at 5 22 04 AM" src="https://user-images.githubusercontent.com/25124304/115477454-8e6d7e80-a261-11eb-92ad-f3a107db9bea.png">
<img width="1060" alt="Screenshot 2021-04-21 at 5 22 13 AM" src="https://user-images.githubusercontent.com/25124304/115477460-90cfd880-a261-11eb-932e-f34eef2dfb3f.png">